enterprise-suite/scripts/lbc.py

```diff
@@ -214,11 +215,40 @@ def check_credentials(creds):
         else:
             printerr('error: check_credentials failed: {}'.format(err))
     finally:
-        return success 
+        return success
 
-def preinstall_check(creds, minikube=False, minishift=False):
+
+
+url_http_timeout=3 # 3 seconds
+installer_url="https://raw.githubusercontent.com/lightbend/console-charts/master/enterprise-suite/scripts/lbc.py"
+warning_display_timeout=2 # 2 seconds
+
+# compare the contents of current running installer and remote installer.
+# if they are different print warning a new installer is present
+def check_new_install_script():
+    try:
+        response=url.urlopen(installer_url, timeout=url_http_timeout)
+        if response == None:
+            return
+        remote_installer_contents=response.read()
+    except url.URLError as e:
+        # if we cannot connect to remote server, ignore for now...
+        return
+
+    # read the contents of the current installer
+    with open(os.path.abspath(__file__)) as f:
+        current_installer_contents = f.read()
+
+    if remote_installer_contents != current_installer_contents:
+        printinfo("\nThe installer you are running is out of date...")
+        printinfo("New installer is available, use the following command to download it")
```

Comment:
How about
A new installer is available. Use the following command to download it.